Wiki/starting/starting-leaving.md

```diff
@@ -0,0 +1,146 @@
+
+```{note}
+It is absolutely fine to create a repository per project or paper with all your codes in. You can then use the README file from the GitHub repository to explain how to reproduce your results. Don't forget to clearly reference everything one might need in addition to this repository. Or you can have a repository per code especially if you envision you'll reuse the same code for other work.
+```
+
+**Configuration files for running the codes and some input files**
+
+In addition to the codes themselves, you need to keep everything that enables someone to run the codes in the same way you have done so. Usually, the most complicated configurations are for climate models. Some climate models will save your configurations in version control repositories (e.g. UM, ACCESS, ACCESS-OM2), in which case you simply need to keep the information on how to retrieve these configurations. Some models don't save your configurations and you need to do it yourself.
+
+For the input files, some inputs are published data in which case you need to keep the reference to this data (including the version). If you have written several codes, the output of a piece of code will be the input of the next piece of code, in which case you do not necessarily need to keep that data,  but you need to describe your workflow.
+
+**Workflow**
+
+This can be a tricky one, as there is not a one-size-fits-all format to save this information. It is fine to write a README file and archive it with other files from the project that need archiving. You can also have a special Github repository just for this README file, or a repository for all your projects with READMEs for each project. Whatever format you choose, it is important for this information to be publicly available (unless your project was restricted) and not a personal note.
+
+This description should clearly describe step by step what someone should do to reproduce your work.
+
+```{warning}
+Information on your PhD or published paper has to be kept for at least 5 years. The time requirements differ slightly depending on institutions and funding bodies.
+```
+
+### To delete, always
+
+You do not need to keep files that are not necessary to reproduce your work:
+
+* log files
+* failed experiments
+* temporary files such as created from successive cdo/nco commands.
+
+```{tip}
+Occasionaly log files might have some information which is relevant to the workflow, still just keeping them makes this information virtually inaccessible to anyone else. The right thing to do in this case is to extract what is useful and add it to the workflow documentation, then getting rid of the files.
+```
+
+### To keep or to delete?
+Some of the data is harder to categorise, and need some careful consideration to decide if it needs to be kept or deleted. Neglecting to do so might result in the deletion of useful data or waste of storage
```

Comment:
```suggestion
Some data is harder to categorise, and deciding whether to keep it or delete it requires careful consideration to avoid. the deletion of useful data or wasting storage space.
```